### PR TITLE
Fixing field permission on create content

### PR DESCRIPTION
**Pull request by bloodbare**

### CODE REVIEW

File: guillotina/api/content.py [hunk 1]
```diff
@@ -224,13 +231,6 @@ async def __call__(self):
 
         await deserializer(data, validate_all=True, create=True)
 
-        # Local Roles assign owner as the creator user
-        get_owner = IGetOwner(obj)
```
Comment (vangheem) [inline guillotina/api/content.py:228]: This seems like a rather big change to not have owner assignment anymore.

File: guillotina/api/content.py [hunk 2]
```diff
@@ -212,6 +212,13 @@ async def __call__(self):
         for behavior in behaviors or ():
             obj.add_behavior(behavior)
 
+        # Local Roles assign owner as the creator user
```
Comment (vangheem) [inline guillotina/api/content.py:215]: Why move this up?

File: guillotina/_settings.py [hunk 3]
```diff
@@ -97,5 +97,12 @@
     "valid_id_characters": string.digits + string.ascii_lowercase + ".-_@$^()+ =",
     "load_catalog": True,
     "catalog_max_results": 50,
+    "managers_roles": {
```
Comment (vangheem) [inline guillotina/_settings.py:100]: can we rename `managers_roles` -> `manager_roles`?

Comment (vangheem) [reply guillotina/_settings.py:100]: Good idea!

**Review by vangheem — CHANGES_REQUESTED** [review]

File: guillotina/api/content.py (hunk 2 above)
Comment (bloodbare) [reply guillotina/api/content.py:215]: If owner is not set before deserialization field permissions are not available as guillotina.Owner on the specific object

**Review by bloodbare — COMMENTED** [review]
the group is called Managers

File: guillotina/_settings.py (hunk 3 above)
Comment (bloodbare) [reply guillotina/_settings.py:100]: the group is called managers

Comment (vangheem) [reply guillotina/_settings.py:100]: But they are `roles` a "Manager" will have. Dunno, just sounds funny to me. You can leave it.

**Review by vangheem — APPROVED** [review]

**Review by masipcat — APPROVED** [review]

**Comment by lferran** [comment]
@bloodbare I'm trying to fix memcached tests. You may want to merge this one on the meantime: https://github.com/plone/guillotina/pull/1102

**Comment by lferran** [comment]
@bloodbare instead of commenting tests, can you merge this one? https://github.com/plone/guillotina/pull/1103

Should be ready now

**Pull request merged**